src/js/components/ui.js

```diff
@@ -234,11 +236,14 @@ const UI = {
 	},
 };
 
-const context = document.getElementById( 'cloudinary-settings-page' );
-
-if ( context ) {
-	// Init.
-	window.addEventListener( 'load', UI._init( context ) );
+const contexts = document.querySelectorAll( '.cld-settings,.cld-meta-box' );
+if ( contexts.length ) {
+	contexts.forEach( ( context ) => {
+		if ( context ) {
+			// Init.
+			window.addEventListener( 'load', UI._init( context ) );
```

Comment:
<!-- highlight specific problems -->
**❗️issue:** @DavidCramer couldn't this be problematic in the edge case of having multiple `context` on the same page?

Reply:
@pereirinha - No, that is the reason. The metabox component could exist a few times on a page.